AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1542,13 +1542,13 @@ petri.com#?#.content_widget:has(> div.widget_content > div.hs-form-insert)
 fort-russ.com#?#.single-container > div.better-studio-shortcode:has(> div.bs-push-noti-inner)
 gatevidyalay.com#?#aside > section.widget-sidebar:has(> div.textwidget > section > form[id^="subscribe-"])
 it-actual.ru##.widget-area > aside.widget > div[class] form[method="post"]
-thebell.io#?#.sidebar-item:has(> .connect)
-thebell.io#?#.sidebar-item:has(> div[id*="ad-sidebar-"])
-thebell.io#?#.sidebar-item:has(> div:contains(weekly newsletter))
+thebell.io##.sidebar
 thebell.io##.follow > .follow__title
-thebell.io##.follow > .follow-form > .follow-form-cell >.follow-form-line
-thebell.io##.follow > .follow-form > .follow-form-cell >.follow__example
-thebell.io#?#.follow > .follow-form > .follow-form-cell:has(> .button)
+thebell.io##.follow > .follow__subtitle
+thebell.io##.follow > .follow-form
+thebell.io##.appbox
+thebell.io##.subs-only__overlay
+thebell.io##article.subs-only:style(height: auto !important; overflow: visible !important;)
```

Comment:
it seems these rules are not required.

Reply:
It's required for subs-only articles (it's not a paywall, it's a free subscription, so no legal problems here). Check out first link in issue.

Reply:
ok. 
`thebell.io##article.subs-only:style(height: auto !important; overflow: visible !important;)` 
=> 
`thebell.io#$#article.subs-only { height: auto !important; overflow: visible !important; }` - AdGuard syntax

Reply:
----
Added directly to filters. Please check for filters update after 1-2 hours.